t/basic.t

```diff
@@ -14,7 +14,7 @@ use_ok("Whim::Core");
 initialize_tests();
 
 diag("Create Whim object");
-my $whim = Whim::Core->new( { data_directory => "$FindBin::Bin/run" } );
+my $whim = Whim::Core->new( { data_directory => $Whim::Core::TRANSIENT_DB } );
```

Comment:
This throws a warning for me:

    Name "Whim::Core::TRANSIENT_DB" used only once: possible typo at t/basic.t line 17.

It goes away if you throw a BEGIN { } around the `use_ok("Whim::Core")` statement.

Reply:
Ah, I missed that. It also goes away if I just `use Whim::Core`. Always been a bit iffy about `use_ok`. The rest of the tests are pointless if they library doesn't load, so it's more fair to think of it as a precondition than a test.

Your project, your preference, Would you rather see `BEGIN{}` and `use_ok`, or is `use` alone sufficient for your testing aesthetics?

Reply:
Ehhhhh let's just throw the BEGIN{} block for now.